cypress/e2e/WebInterface/Measure/VersionDraftMeasure/DraftMeasure.cy.ts

```diff
@@ -0,0 +1,179 @@
+import { OktaLogin } from "../../../../Shared/OktaLogin"
+import { MeasuresPage } from "../../../../Shared/MeasuresPage"
+import { MeasureGroupPage } from "../../../../Shared/MeasureGroupPage"
+import { CreateMeasurePage } from "../../../../Shared/CreateMeasurePage"
+import { MeasureCQL } from "../../../../Shared/MeasureCQL"
+import { Utilities } from "../../../../Shared/Utilities"
+import { EditMeasurePage } from "../../../../Shared/EditMeasurePage"
+import { CQLEditorPage } from "../../../../Shared/CQLEditorPage"
+import { Header } from "../../../../Shared/Header"
+import { contains } from "cypress/types/jquery"
+
+let MeasuresPageOne = ''
+let updatedMeasuresPageName = ''
+let randValue = (Math.floor((Math.random() * 1000) + 1))
+let newMeasureName = ''
+let newCqlLibraryName = ''
+let ratioMeasureCQL = MeasureCQL.ICFCleanTest_CQL
+let cohortMeasureCQL = MeasureCQL.CQL_For_Cohort
+
+describe('Draft and Version Validations -- add and cannot create draft of a draft that already exists tests', () => {
```

Comment:
lets add some skips to these test cases, you can do it at the describe. Maybe put a comment about which FLAG we need to be removed for these test cases to be active

Reply:
skipped all new tests and the version flag's name is measureVersioning -- I confirmed this with Prateek